Source/AliveLibAE/Input.cpp

```diff
@@ -1992,7 +1992,7 @@ void InputObject::Update_45F040()
         {
             const DWORD command = (*field_30_pDemoRes)[field_34_demo_command_index++];
             field_3C_command = command >> 16;
-            field_40_command_duration = sGnFrame_5C1B84 + command & 0xFFFF;
+            field_40_command_duration = (sGnFrame_5C1B84 + command) & 0xFFFF;
```

Comment:
oh, this one actually isn't right, its supposed to be sGnFrame_5C1B84 + (command & 0xFFFF); as you can see the other 16bits are being used as the command above, and these are the time/duration.

Reply:
I can fix it, but it's not the same behavior as prior to my PR . `+` has higher precedence than `&`.

Reply:
yeah its a bug - I do wonder how demo play back was working now though 😂